test-manual/helius-laser/sh/03_create-config.sh

```
@@ -0,0 +1,31 @@
+if [[ -n "${TRITON_API_KEY:-}" ]]; then
+    TEMPLATE_FILE="$CONFIG_DIR/triton-config-template.toml"
+    TRITON_API_KEY="$TRITON_API_KEY"
+    HELIUS_API_KEY="$HELIUS_API_KEY"
+    sed "s/<helius-api-key>/$HELIUS_API_KEY/g" "$TEMPLATE_FILE" |  \
+    sed "s/<triton-api-key>/$TRITON_API_KEY/g"  \
+    > "$OUTPUT_FILE"
+else
+    TEMPLATE_FILE="$CONFIG_DIR/helius-config-template.toml"
+    HELIUS_API_KEY="$HELIUS_API_KEY"
+    sed "s/<helius-api-key>/$HELIUS_API_KEY/g" "$TEMPLATE_FILE" |  \
+    > "$OUTPUT_FILE"
```

Shellcheck (0.11.0)

[error] You can't have | between this redirection and the command it should apply to.

(SC2189)